specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -9095,6 +9095,14 @@
           "items": {
             "$ref": "#/definitions/RecoveryPlanGroup"
           }
+        },
+        "providerSpecificInput": {
```

Comment:
Are these not returned as part of the resource?

Reply:
These are returned but within RecoveryPlanProviderSpecificDetails, since we may output some other provider specific details in future which may not be part of providerSpecificInput.